src/components/share/links/FacebookLink.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react'
 import ShareLink from './ShareLink'
 
-const FacebookLink = ({ slug, title, styles }) => {
+const FacebookLink = ({ pageConfig }) => {
+  const { host, port, slug, title } = pageConfig
+
   const fbLink = 'http://www.facebook.com/sharer.php',
-    shareSite = encodeURIComponent(`https://codemate.com${slug}`),
+    shareSite = encodeURIComponent(`https://${host}:${port}${slug}`),
```

Comment:
The Netlify builds broke due to the following reason: 

```
GraphQL Error: There was an error while compiling your site's GraphQL queries.
Invariant Violation: GraphQLParser: Unknown field `host` on type `Site`. Source: document `BlogPostBySlug` file: `GraphQL request`.
```

I would bet that these will also break "real" sites (think when this blog gets published) as well, although locally on `localhost:4000` they will work fine.